lips/lip-8.md

```diff
@@ -43,11 +51,18 @@ All requests between VASPs are structured as [`CommandRequestObject`s](https://l
         "fund_pull_pre_approval": {
             "address": "lbr1pgfpyysjzgfpyysjzgfpyysjzgf3xycnzvf3xycsm957ne",
             "biller_address": "lbr1pg9q5zs2pg9q5zs2pg9q5zs2pg9skzctpv9skzcg9kmwta",
-            "funds_pre_approval_id": "lbr1pg9q5zs2pg9q5zs2pg9q5zs2pgyqqqqqqqqqqqqqqspa3m_7b8404c986f53fe072301fe950d030de"
-            "expiration_timestamp": 72322, 
-            "max_cumulative_amount": {
-                "amount": 1000,
-                "currency": "LBR"
+            "funds_pre_approval_id": "lbr1pg9q5zs2pg9q5zs2pg9q5zs2pgyqqqqqqqqqqqqqqspa3m_7b8404c986f53fe072301fe950d030de",
+            "scope": {
```

Comment:
what's the value of having another data structure? could we amend the PR description or is there a tracking doc for this?